### ISSUE TITLE
npm run doc

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#438]
should be in CI (frontend builds)

**Comment by fel115** [comment]
I tried to build the storybook statically on my local machine using the command `npm run build-stroybook`, but I get following error:
```
ERR! Error: Unable to index ./src/generated/docs/docs/modules/features_map_planning_store_MapStoreTypes.mdx
ERR!     at StoryIndexGenerator.getIndex (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:17:3457)
ERR!     at async extractStoriesJson (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:10:4413)
ERR!     at async Promise.all (index 3)
ERR!     at async buildStaticStandalone (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:31:1793)
ERR!     at async withTelemetry (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:35:3422)
ERR!     at async build (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/cli/dist/generate.js:434:1058)
ERR!     at async Command.<anonymous> (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/cli/dist/generate.js:437:125)
ERR!  MultipleIndexingError: Unable to index ./src/generated/docs/docs/modules/features_map_planning_store_MapStoreTypes.mdx
ERR!     at StoryIndexGenerator.getIndex (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:17:3457)
ERR!     at async extractStoriesJson (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:10:4413)
ERR!     at async Promise.all (index 3)
ERR!     at async buildStaticStandalone (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:31:1793)
ERR!     at async withTelemetry (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:35:3422)
ERR!     at async build (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/cli/dist/generate.js:434:1058)
ERR!     at async Command.<anonymous> (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/cli/dist/generate.js:437:125) {
ERR!   indexingErrors: [
ERR!     IndexingError: Unexpected character `` ` `` (U+0060) before name, expected a character that can start a name, such as a letter, `$`, or `_`
ERR!         at /home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:13:2054
ERR!         at async Promise.all (index 100)
ERR!         at async Promise.all (index 0)
ERR!         at async StoryIndexGenerator.updateExtracted (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:13:1601)
ERR!         at async StoryIndexGenerator.ensureExtracted (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:13:2357)
ERR!         at async StoryIndexGenerator.initialize (/home/smod/Bachelorarbeit/GitHub/PermaplanT/frontend/node_modules/@storybook/core-server/dist/index.js:13:1528) {
ERR!       importPaths: [Array]
ERR!     }
ERR!   ]
ERR! }
fatal error: all goroutines are asleep - deadlock!
```

Does anyone have an idea what is causing this error?

**Comment by markus2330** [comment]
Thanks for reporting this problem!

@Bushuo recently mentioned this problem somewhere.

Maybe @4ydan can fix it?

Btw. there is also #439, it is probably the same?

**Comment by Bushuo** [comment]
@markus2330 I am working on a fix in #466. Will try to get it ready for a review today. 
Sorry for the confusion.

**Comment by markus2330** [comment]
Thx! I merged #466, so everything should now be ready for @4ydan to actually let it check in CI?
